### PR TITLE
chore: adjust coverage in jacoco

### PR DESCRIPTION
**Pull request by uziasferreirazup**
### Checklist

Please, check if these important points are met using `[x]`:

- [ ] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [ ] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [ ] I am willing to follow-up on review comments in a timely manner.
- [ ] I have made the documentation changes or I created an issue explaining how to document this change on [Docs issues](https://github.com/ZupIT/beagle-docs/issues). Please link the issue here:

[PR Guide]: https://github.com/ZupIT/beagle/blob/main/doc/contributing/pull_requests.md

### CODE REVIEW

File: backend/build.gradle [hunk 1]
```diff
@@ -70,9 +72,29 @@ allprojects {
...
+    additionalSourceDirs.setFrom files(subprojects.sourceSets.main.allSource.srcDirs)
+    sourceDirectories.setFrom files(subprojects.sourceSets.main.allSource.srcDirs)
+    classDirectories.setFrom files(subprojects.sourceSets.main.output)
+
+    executionData.setFrom project.fileTree(dir: '.', include: '**/build/jacoco/test.exec')
+
+    reports {
+        xml.enabled true
+        html.enabled false
+    }
```
Comment (hernandazevedozup) [inline backend/build.gradle:99]: Can you separate this code in a mergedCoverage.gradle file?

Comment (hernandazevedozup) [reply backend/build.gradle:99]: also the apply plugin may work on the new file too

**Review by paulomeurerzup — APPROVED** [review]

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by paulomeurerzup — APPROVED** [review]

**Comment by igorgiumellizup** [comment]
Android tests ok:
image

**Review by igorgiumellizup — APPROVED** [review]

**Review by matheusribeirozup — APPROVED** [review]

**Pull request merged**